deliver/lib/deliver/options.rb

```diff
@@ -181,6 +181,11 @@ def self.available_options
                                      description: "Clear all previously uploaded screenshots before uploading the new ones",
                                      is_string: false,
                                      default_value: false),
+        FastlaneCore::ConfigItem.new(key: :skip_unsupported_languages,
```

Comment:
I was asking myself the same thing. I originally wasn't having it crash cause that would be annoying but this is something that can easily get lost in the logs. I wasn't sure if this made things clear or murkier 🤔  But you are correct, setting it to false does ask it to crash. 

I personally don't do a lot of localized apps so I don't _really_ know if this is a big issue or not 🙈